lib/streamlit/elements/multiselect.py

```diff
@@ -93,11 +93,11 @@ def multiselect(
         check_callback_rules(self.dg, on_change)
         check_session_state_rules(default_value=default, key=key)
 
-        options = ensure_iterable(options)
+        opt = ensure_indexable(options)
```

Comment:
Not a big deal, but it might value either a comment or a better name to ensure the input is iterable and indexable.

Reply:
Indexable pretty straightforwardly implies it should be iterable to me, unsure what a useful comment or better name would be.

Reply:
Hmmm...perhaps you're right and I am overthinking it.